src/main/java/com/google/sps/data/CalendarDataResponse.java

```diff
@@ -0,0 +1,54 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+import java.util.*;
+
+/** Class containing the response to be converted to Json. */
+public final class CalendarDataResponse {
+
+  final long startDay;
```

Comment:
What are the units of `startDay`?

Reply:
Changed to int. startDay is a value between 0 and 6 giving the day of the week on which to start displaying on the client side. `Resolved`